clients/githubrepo/contributors_e2e_test.go

```diff
@@ -24,11 +25,18 @@ import (
 
 	"github.com/ossf/scorecard/v5/clients"
 	"github.com/ossf/scorecard/v5/clients/githubrepo/roundtripper"
+	"github.com/ossf/scorecard/v5/internal/gitfile"
 	"github.com/ossf/scorecard/v5/log"
 )
 
 var _ = Describe("E2E TEST: githubrepo.contributorsHandler", func() {
 	var contribHandler *contributorsHandler
+	var fileHandler io.ReadCloser
+	repoURL := Repo{
+		owner:     "ossf",
+		repo:      "scorecard",
+		commitSHA: clients.HeadSHA,
+	}
```

Comment:
Flagging this initially, as I tend to prefer repos that need the files from `ossf/scorecard`, as they take longer to process in tests.

Reply:
this increases the test time by 50% or so.

If I make a new e2e test repo, will you switch the test to use it?